src/pages/index.js

```diff
@@ -143,30 +207,26 @@ previewImageModal.setEventListeners();
 function createCard(data) {
   const card = new Card(
     data,
-    cardSelector,
-    handleImageClick,
-    handleDeleteCard,
-    handleLike
+    "#card-template",
+    openPreviewModal,
+    handleDeleteAction
   );
   return card.getView();
 }
 
-function renderCard(carddata) {
-  const cardInstance = new Card(
-    carddata,
-    cardSelector,
-    handleImageClick,
-    handleDeleteCard,
-    handleLike
-  );
-  return cardInstance.getView();
+function handleDeleteAction(card) {
+  cardDeleteModalModal.open();
+  cardDeleteModal.setSubmitAction(() => {
+    card._handleDeleteAction();
+    cardDeleteModal.close();
+  });
 }

```

Comment:
Please, delete `createCard` and `handleDeleteAction`  because you don't use them